tests/headingcommand.js

```diff
@@ -217,12 +204,12 @@ describe( 'HeadingCommand', () => {
 				);
```

Comment:
Do you still refer to https://github.com/ckeditor/ckeditor5-heading/pull/84#discussion_r128191087?